### DISCUSSION TITLE
Alert overview empty

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#8958] [Unsupported Versions] [answered]
Hello!

If I switch to "Group By Sensor, Source IP/Port, Destination IP/Port, Name" in the Alert view of SOC it's just empty (even though there are alerts):

![image](https://user-images.githubusercontent.com/81571847/196980055-081622ad-e5ea-4436-9356-7fd2364fae25.png)

I noticed that a few days ago in SO 2.3.170 and it's still an issue in 2.3.180. How can I solve that? Thanks in advance!

Cheers, Ben

**Comment by dougburks** [comment, marked as answer]
This most likely means that one of the fields is not aggregatable in your deployment. Try clicking the trash can icon next to `network.community_id` to see if that helps.

**Reply by ben-sec** [reply]
That helped!
